subjects/gaming/stealth-boom/README.md

```diff
@@ -1,82 +1,98 @@
-    - If the player leaves the field of view for a certain time, the Guard goes back to patrol mode.
-  - Drones AI:
-    - Drones should be able to patrol around the map;
-    - A light color should determine the state of the drone (Blue for patrolling, Red for chasing the player);
-    - Once the player crosses the drone camera, the drone light turns red and the drone enters chasing mode;
-    - When a drone is in chasing mode, all the guards on the area are alerted, and should enter chasing mode as well;
-    - When the player is out of the drone sight, the drone turns back to patrol mode;
-    - The sight radius should be inferior on the drones that on the guards.
-  - Camera AI:
-    - Cameras should be placed on walls;
-    - Cameras should have the same light sign as the drone, so when the player is in the camera sight, the camera light turns red and all Guards enter in chasing mode;
-    - Like the Drones, Cameras warn guards, whenever the player passes through the camera field of view;
-    - Some Cameras should lock access of certain areas of the map (for example, close doors) when the player is detected by that camera.
-
-- Drones, Guards and Cameras should have sounds effect whenever they change from chase to patrol mode, as well as the other way around.
-
-- All AI should be implemented using Behavior Trees except for the Camera.
-
-- The player mission is up to you, either it can be some task to fix, kill all guards without getting caught or collect documents... Whatever you choose, the player should have enemies on his way to divert him away from his objective.
+  - Change the mouse sensitivity.
+  - Invert the mouse vertical axis.
+
+- Map/Level.
+
+  - This map should contain places for the player to hide from enemies by `Ducking`, `Hiding behind walls and props`.
+  - Pickable ammunition around the map.
+  - Pickable Health around the map.
+
+- Player:
 
+  - Walk and run animations.
+  - Shoot animation.
+  - Duck animation.
+  - Melee attack animation.
+  - Blood particles when hit.
+  - A health bar that should go down whenever the player gets damaged. When the player dies, he has the choice to either quit the game, go back to the main menu or start over.
+  - The player mission is up to you, it can be some task to fix, kill all enemies without getting caught or collect documents... Whatever you choose, the player should have enemies on his way to divert him away from his objective.
   - When the player successfully completes his mission, a pop up should appear saying that the mission is completed.
 
-- The player has a health bar that should go down whenever the player gets shot. When the player dies, he has the choice to either quit the game, go back to the main menu or start over.
+- Gun mechanics:
+
+  - a widget displaying the number of bullets.
+  - when the number of bullets is 0 the player is unable to shoot.
+  - shooting should have a sound effect and visual effect.
+  - Bullets visual impact on walls.
```

Comment:
```
  - A widget should display the current number of bullets available to the player.
  - The player should be unable to shoot when the bullet count reaches zero.
  - Shooting should include both a sound effect and a visual effect.
  - Bullet impacts on walls should be visually represented.
```

Reply:
There is still a typo in this section, and I noticed you didn't change to the version I proposed as well, do you think it's better as it is?

Reply:
i fixed the typo, i didn't keep the old version, i used your version to refine it.
mainly to keep the sentences concise and consistent with the rest of the subject.

Reply:
i fixed the typo, i didn't keep the old version, i used your version to refine it.
mainly to keep the sentences concise and consistent with the rest of the subject.